### PR TITLE
Package-ify scratchr2_translations

### PR DESCRIPTION
**Pull request by mewtaylor**
So that it can be npm installed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "scratchr2_translations",
+  "version": "1.0.0",
+  "description": "scratchr2_translations ======================",
```
Comment (rschamp) [inline package.json]: Doesn't matter too much but `"scratchr2_translations ======================"`?

**Comment by rschamp** [comment]
LGTM

File: package.json (hunk 1 above)
Comment (mewtaylor) [inline package.json]: It's a solid, descriptive name and I stand by it.

Comment (mewtaylor) [inline package.json]: (just kidding. I should change that.)

Comment (rschamp) [inline package.json]: Suggestion: `"scratchr2_translations ====================== true"`. Just to be extra sure.

Comment (mewtaylor) [inline package.json]: haha part of me reeeeeeeeeally wants to do that.

**Comment by mewtaylor** [comment]
@rschamp I'm actually going to close this an re-open later. I'm going to add a node executable to this package called `export2json` that will handle the conversion of po to json (and any associated tests) so that it can be easily called as part of a build. Does that sound good to you?

**Comment by rschamp** [comment]
That sounds like it would be more flexible as a standalone tool `po2json`, and we could just use it with these po files (or any others we need).

**Comment by mewtaylor** [comment]
it'll actually use something already called `po2json` that exists. This is more just to do it in batch, and to handle the weird file structure with `LC_MESSAGES` that is repo-specific here. Seemed to me that handling those things, if this is to be a package, should be handled on the package's side of things, rather than on the dependent's side of things.

**Comment by rschamp** [comment]
To me it seems like the dependent's responsibility to turn these po files into the format and location it needs (and `LC_MESSAGES` is a standard gettext thing).  If we change how we need these to be converted or whatever, that would also probably be a change we'd want to track in the thing using the po files, not here.  I don't feel that strongly about it though.

**Comment by mewtaylor** [comment]
:+1:

**Pull request merged**